src/OpenIddict.Security/project.json

```diff
@@ -0,0 +1,22 @@
+﻿{
+    "version": "1.0.0-*",
+
+    "description": "Security headers module for OpenIddict.",
+
+    "dependencies": {
+        "OpenIddict.Core": "1.0.0-*",
+        "NWebsec.Middleware": {
+            "type": "build",
+            "version": "1.0.0-gamma-custom-*"
```

Comment:
Please note I renamed the package and changed the version in a really recent commit.